### PR TITLE
Minor Doc Changes

### PR DESCRIPTION
**Pull request by KyouBrayden**
See commits.

### CODE REVIEW

File: build/README.md [hunk 1]
```diff
@@ -4,184 +4,186 @@ How to build
...
+This method will build a Linux binary.
 
 * On Ubuntu 22.04, install the following packages:
-   * `build-essential`
-   * `libasound2-dev`
-   * `libx11-dev`
-   * `libxpm-dev`
-   * `mesa-common-dev`
-   * `libgstreamer1.0-dev`
-   * `libgstreamer-plugins-base1.0-dev`
```
Comment (ktabata) [inline build/README.md:112]: We need these lines.

File: build/README.md [hunk 2]
```diff
@@ -4,184 +4,186 @@ How to build
...
+
+Run the application from your device or an emulator.
+
+## Web
+This method will utilise the Emscripten framework to build a web distribution.
+
+* Prerequisites
+  * Use a UNIX-like environment such as MSYS2, Linux, or macOS.
+  * Ensure you can access `make` and `python`.
+  * Install `Emscripten` using the `emc` command.
```
Comment (ktabata) [inline build/README.md:82]: Use `emsdk` instead of `emc`

File: build/README.md [hunk 3]
```diff
@@ -4,184 +4,186 @@ How to build
...
+## Raspberry Pi Binary
+This method will build a Raspberry Pi binary.
 
 * On Raspberry Pi OS, install the following packages:
-   * `libasound2-dev`
-   * `libx11-dev`
-   * `libxpm-dev`
-   * `mesa-common-dev`
-   * `libgstreamer1.0-dev`
-   * `libgstreamer-plugins-base1.0-dev`
```
Comment (ktabata) [inline build/README.md:133]: We need these lines.

**Review by ktabata — CHANGES_REQUESTED** [review]
Overall, I thought it was very nice and easy to see. Thank you!

**Comment by KyouBrayden** [comment]
@ktabata Please let me know if these corrections are sufficient.
Apologies for the mistake! I forgot to change them back after copy+paste.

**Review by ktabata — APPROVED** [review]
lgtm thank you!

**Pull request merged**